plugins/kotlin/idea/src/org/jetbrains/kotlin/idea/inspections/UnusedUnaryOperatorInspection.kt

```diff
@@ -56,7 +82,29 @@ class UnusedUnaryOperatorInspection : AbstractKotlinInspection() {
             prefixExpression.replace(baseExpression)
         }
     }
+
+    private class MoveUnaryOperatorToPreviousLineFix : LocalQuickFix {
```

Comment:
Probably worth to make it a `HighPriorityAction`?

Reply:
74a9c0d03e5ebf96e19e384be3d1ad6d13b397c4